core/src/test/resources/com/google/googlejavaformat/java/testdata/B20128760.output

```diff
@@ -89,19 +89,13 @@ class B20128760 {
   {
     Stream<ItemKey> itemIdsStream =
         stream(members)
-            .flatMap(
-                m ->
-                    m.getFieldValues()
+            .flatMap(m -> m.getFieldValues()
```

Comment:
IMO, lines like this would look better with a break after the arrow `->`, but no break before the arrow. So something like this:
```java
...
.flatMap(m ->
    m.getFieldValues()
        .stream()
        .filter(...)
        ...);